docker/Dockerfile

```diff
@@ -10,5 +10,10 @@ RUN ./pkg_install.sh py-numpy
 RUN ./pkg_install.sh opencv@2.4.13.2 +highgui +imgproc +python
 RUN ./pkg_install.sh image-magick
 
+RUN touch /home/drgscl/.zshrc.mod && echo '\
```

Comment:
The original `entrypoint.sh` script will check the existence of `/home/drgscl/.zshrc.mod` and source it after loading all existing modules via `spack load`. This is the place to swap incompatible modules with the intended version.